### DISCUSSION TITLE
Force 5.1 surround support

### DISCUSSION THREAD

**Discussion by Primajin** [ChristopherKlay/StadiaEnhanced#229] [Ideas & Requests]
It would really great when I could force 5.1 surround sound. I have a 7.1 surround device but apparently Chrome doesn't care.

**Comment by ChristopherKlay** [comment]
Chrome *should* [support 5.1 via the website](https://support.google.com/stadia/answer/9607891#zippy=%2Crequirements-for-surround-sound) just fine.

Is 5.1 working in Chrome in general? You can check it [here](https://www2.iis.fraunhofer.de/AAC/multichannel.html). (:

**Comment by Primajin** [comment]
You are right! Windows inadvertently set my device to Stereo although it is 7.1. After I changed it back in the system settings it works now!

![image](https://user-images.githubusercontent.com/1742115/161417369-74e9c6ec-efd2-4682-8c36-b21f5dc8042a.png)

**Reply by Primajin** [reply]
This topic can be closed as obsolete if you like
